plugins/obs-ffmpeg/ffmpeg-mux/ffmpeg-mux.c

```diff
@@ -27,8 +27,16 @@
 #include <stdlib.h>
 #include "ffmpeg-mux.h"
 
+#include <obs-module.h>
```

Comment:
Is obs-module.h needed?

Reply:
You're right it's not! I think I needed it in an earlier version of the code but I no longer need it.